main.tf

```diff
@@ -1,7 +1,7 @@
 provider "aws" {
   version = "2.33.0"
 
-  region = var.aws_region
+  region = var.aws_west
```

Comment:
This change broke the project. `aws_west` is not defined. But `aws_region` is defined in `variables.tf`. Here's what I get running the Terraform Cloud tutorial using the current state of the project:

```
 Terraform v0.12.29
Configuring remote state backend...
Initializing Terraform configuration...
2020/07/31 17:10:09 [DEBUG] Using modified User-Agent: Terraform/0.12.29 TFC/3ad8313ada

Error: Reference to undeclared input variable

  on main.tf line 4, in provider "aws":
   4:   region = var.aws_west

An input variable with the name "aws_west" has not been declared. This
variable can be declared with a variable "aws_west" {} block.
```